### DISCUSSION TITLE
Lawnchair 14 Dev : Market or GitHub debug apk ?

### DISCUSSION THREAD

**Discussion by pokemaster974** [LawnchairLauncher/lawnchair#3982] [Q&A] [answered]
Hi, downloading last 14-dev debug apk gives me 2 apk : Lawnchair 14 Dev github debug and Lawnchair 14 Dev market debug.
What's the difference between these 2 apk ?
Regards

**Comment by MrSluffy** [comment, marked as answer]
Only the package name, market one is for playstore releases etc.

**Reply by pokemaster974** [reply]
Thanks for the reply.
